moj-node/test/repositories/SearchRepository.spec.js

```diff
@@ -0,0 +1,288 @@
+const searchRepository = require('../../server/repositories/search');
+
+describe('searchRepository', () => {
+  describe('#find', () => {
+    describe('When called with a search query', () => {
+      it('should return the results', async () => {
+        const mockResults = [
+          {
+            _id: '1',
+            _source: {
+              title: ['First Item'],
+              summary: ['First item summary'],
+            },
+          },
+          {
+            _id: '2',
+            _source: {
+              title: ['Second Item'],
+              summary: ['Second item summary'],
+            },
+          },
+          {
+            _id: '3',
+            _source: {
+              title: ['Third Item'],
+              summary: ['Third item summary'],
+            },
+          },
+        ];
+
+        const client = createClient({
+          body: { hits: { hits: mockResults } },
+        });
+
+        const repository = searchRepository(client);
+
+        const response = await repository.find({
+          query: 'Test Query',
+          prison: 'HMP Development',
+        });
+
+        expect(client.post.callCount).to.equal(
```

Comment:
You probably want to test that that the client was called with the query provided?

Reply:
Added assertions to check for the query in the request body